README.md

```diff
@@ -267,6 +267,8 @@ The Device Repository supports three payload codecs to be defined:
 2. Downlink encoder: decodes a JSON object into binary data downlink
 3. Downlink decoder: decodes an encoded binary data downlink back into a JSON object (must be symmetric with the downlink encoder)
 
+Recomendation. For a better user experience, it is better that decoders indicates the units used.
```

Comment:
```suggestion
**Note:** We recommend using measure units in payload decoders for a better user experience.
```

Reply:
I changed file, also I added the full variable name recommendation like: 

**Note:** We recommend using measure units and full variable names in payload decoders for a better user experience.